setup.cfg

```diff
@@ -12,7 +12,7 @@ author_email = ramanathana@anl.gov
 [options]
 packages = find:
 install_requires =
-    pydantic==1.10.6
+    pydantic>=1.10.6
```

Comment:
This will cause an error if pydantic is version 2 or greater. Still need to update it. Maybe best to pin the exact version like it the requirements.txt for now. `pydantic==1.10.9`

Reply:
OK, sounds good.